CMakeLists.txt

```diff
@@ -85,7 +85,9 @@ if(APPLE)
 endif()
 message(STATUS "Using find_package to locate Vulkan")
 find_package(Vulkan)
-find_package(VulkanHeaders)
+find_package(VulkanHeaders REQUIRED CONFIG)
+get_target_property(VulkanHeaders_INCLUDE_DIR Vulkan::Headers INTERFACE_INCLUDE_DIRECTORIES)
+get_target_property(VulkanRegistry_DIR Vulkan::Registry INTERFACE_INCLUDE_DIRECTORIES)
```

Comment:
Why not update the code that relies on `VulkanHeaders_INCLUDE_DIR` instead? 
Or should `Vulkan-Headers` "export" VulkanHeaders_INCLUDE_DIR?